### PR TITLE
Action on map

### PR DESCRIPTION
**Pull request by mun625**
implemented "action - reducer - state" in right form.
this PR is to unify applying "action - reducer - state" style.

### CODE REVIEW

File: ios/Goober.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	};
 	objectVersion = 46;
 	objects = {
+
```
Comment (mun625) [inline ios/Goober.xcodeproj/project.pbxproj]: Can anyone explain why this file is changed???

**Comment by elegant651** [comment]
LGTM

File: shared/components/map.js [hunk 2]
```diff
@@ -23,16 +23,13 @@ const styles = StyleSheet.create({
 
 export default class Map extends Component {
   render() {
+    console.log('np man');
```
Comment (yuda1124) [inline shared/components/map.js]: I think this line should be removed

File: shared/reducers/map.js [hunk 3]
```diff
@@ -10,6 +15,11 @@ export default function map(state = initialState, action = {}) {
     return {
       ...state
     };
+  case types.onLocationChange:
+  	console.log('I am in broooo');
```
Comment (yuda1124) [inline shared/reducers/map.js]: kk same!

File: shared/components/map.js (hunk 2 above)
Comment (mun625) [inline shared/components/map.js]: who did this man?

**Comment by yuda1124** [comment]
LGTM

**Pull request merged**